### PR TITLE
Persistent Bcast API implementation [For testing]

### PR DESCRIPTION
**Pull request by akhillanger**

### CODE REVIEW

**Comment by akhillanger** [comment]
test:jenkins/ch3/most
test:jenkins/ch4/most

**Comment by akhillanger** [comment]
test:jenkins/ch3/mxm

**Comment by akhillanger** [comment]
test:jenkins/ch3/most
test:jenkins/ch4/most

**Comment by akhillanger** [comment]
test:jenkins/ch3/sock

**Comment by akhillanger** [comment]
test:jenkins/ch3/sock

**Comment by akhillanger** [comment]
test:jenkins/ch3/sock

**Comment by akhillanger** [comment]
test:jenkins/ch3/sock

File: src/include/mpi.h.in [hunk 1]
```diff
@@ -2196,7 +2200,6 @@ int PMPI_T_pvar_get_index(const char *name, int var_class, int *pvar_index) MPIC
 int PMPI_T_category_get_index(const char *name, int *cat_index) MPICH_API_PUBLIC;
 /* End Skip Prototypes */
 
-
```
Comment (wesbland) [inline src/include/mpi.h.in]: Whitespace only changes should either **be removed** or done separately.

File: src/mpi/coll/bcast/bcast_init.c [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    return mpi_errno;
+  fn_fail:
+    goto fn_exit;
+}
+
+#endif
+
+#undef FUNCNAME
+#define FUNCNAME MPIX_Bcast_init
+#undef FCNAME
```
Comment (wesbland) [inline src/mpi/coll/bcast/bcast_init.c]: This is missing

    #define FCNAME MPL_QUOTE(FUNCNAME)

**Comment by wesbland** [comment]
I added a fixup patch that lets it build on my local system. Let's see what happens here.

test:jenkins/ch3/most
test:jenkins/ch4/most

**Comment by akhillanger** [comment]
test:jenkins/ch3/most
test:jenkins/ch4/most

**Pull request closed without merging**